packages/eslint-config/package.json

```diff
@@ -27,20 +27,15 @@
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-jest": "^27.2.1",
     "eslint-plugin-promise": "^6.1.1",
-    "eslint-plugin-sort-class-members": "^1.17.1"
+    "eslint-plugin-sort-class-members": "^1.17.1",
+    "prettier": "3.2.5",
+    "eslint-config-prettier": "^8.8.0 || ^9.1.0",
+    "eslint-plugin-prettier": "^5.1.3"
```

Comment:
https://stackoverflow.com/a/76625797/22554364

prettier with version >3 is compatible only with >5 eslint-plugin-prettier